package.json

```diff
@@ -20,22 +20,23 @@
         "precommit": "npm run eslint && npm run stylelint",
         "test": "jest"
     },
-    "dependencies": {
+    "devDependencies": {
         "clean-css-cli": "4.2.1",
         "eslint": "5.1.0",
         "eslint-plugin-jest": "22.1.2",
-        "node-sass": "4.9.4",
-        "stylelint": "9.3.0",
-        "uglify-js": "2.8.29"
-    },
-    "devDependencies": {
         "gulp": "3.9.1",
         "gulp-sass": "4.0.1",
         "gulp-stylelint": "7.0.0",
         "gulp-watch": "5.0.0",
         "husky": "0.14.3",
         "jest": "23.6.0",
+        "node-sass": "4.9.4",
+        "react": "16.8.1",
```

Comment:
I&nbsp;don't really feel like react should be&nbsp;added just yet in&nbsp;this&nbsp;PR, given the&nbsp;discussion in&nbsp;https://github.com/mdn/sprints/issues/967.